.github/workflows/dlp-pipelines.yml

```diff
@@ -254,6 +254,81 @@ jobs:
           done
           echo "# records in input CSV file are: $rc_orig."
           echo "Verified number of rows in ${{env.INPUT_FILE_NAME}}_pub_sub: $row_count."
+  
```

Comment:
Also, were these changes tested on cases where we were getting duplicates to ensure it works there? You can refer to the [experiments](https://docs.google.com/spreadsheets/d/1xxwWLYGsPNMfpZ6CquaJ7_DVpJa6lIJ5Xhm1G8BS0AQ/edit?resourcekey=0-ExzIpDApP_RwZmIR0dzX-A#gid=0) done previously for the same.

If already done, might be worth mentioning under TESTED.

Reply:
@chitara-01 Average time taken by entire workflow has increased because one of the job among inspection, de-identification and de-identification-streaming-write is waiting for the runner to finish others. We need to create one more self-hosted runner to make it efficient

Reply:
> Also, were these changes tested on cases where we were getting duplicates to ensure it works there? You can refer to the [experiments](https://docs.google.com/spreadsheets/d/1xxwWLYGsPNMfpZ6CquaJ7_DVpJa6lIJ5Xhm1G8BS0AQ/edit?resourcekey=0-ExzIpDApP_RwZmIR0dzX-A#gid=0) done previously for the same.
> 
> If already done, might be worth mentioning under TESTED.

Will test this when comparing throughput and reliability improvements. For easy tracking, noting it down under PR description